hw/display/apple_h12p.c

```diff
@@ -243,8 +272,12 @@ static const VMStateDescription vmstate_apple_h12p = {
 };
 
 static Property apple_h12p_props[] = {
-    DEFINE_PROP_UINT32("width", AppleH12PState, width, 480),
-    DEFINE_PROP_UINT32("height", AppleH12PState, height, 680),
+    // iPhone 4/4S
+    DEFINE_PROP_UINT32("width", AppleH12PState, width, 640),
+    DEFINE_PROP_UINT32("height", AppleH12PState, height, 960),
+    // iPhone 11
+    //DEFINE_PROP_UINT32("width", AppleH12PState, width, 828),
+    //DEFINE_PROP_UINT32("height", AppleH12PState, height, 1792),
```

Comment:
Is this really required? It works with the old resolution value too.

<img width="592" alt="Screenshot 2023-07-10 at 21 51 46" src="https://github.com/ChefKissInc/qemu-t8030/assets/30368284/6efc23e8-d0e8-4c76-86fd-df95c43ed5d8">

Reply:
According to a google search I made a week ago, having a value there that makes libMobileGestalt complain will result e.g. in crashing apps.
So I guess it would be best to set it to a value, which is expected by libMobileGestalt.